WordPress/Classes/Models/SiteInformation.swift

```diff
@@ -0,0 +1,11 @@
+struct SiteInformation {
```

Comment:
At first glance, this resembles [`SiteCreationParams`](https://github.com/wordpress-mobile/WordPress-iOS/blob/f7d90c8d8711314ffcf1574610a514dec99b1bd8/WordPress/Classes/Services/SiteCreationService.swift#L20-L32).

We'll need to keep this in mind as we proceed with #10429.

Reply:
🤔 I agree. `SiteCreationParams` seems to have a role more like our `SiteCreator` to me though, don't you think?